btcpay-admin.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+query()
+{
+    docker exec $(docker ps -a -q -f "name=postgres_1") psql -U postgres -d btcpayservermainnet -c "$*"
+}
+
+case "$1" in
+    reset-u2f)
+        query "SELECT * FROM \"U2FDevices\""
+        query "UPDATE public.\"AspNetUsers\" SET \"TwoFactorEnabled\"=false WHERE upper('\$1') = \"NormalizedEmail\""
```

Comment:
I don't understand, whois `\$1` ?

Reply:
This is wrong, it should  be $2 so that user specifies which user should have its multifactor reset

Reply:
I am unsure as the doc says there is no parameter.

Reply:
I was using the SQL you gave me lol. Anyway yeah feel free to add or tweak whatever queries and other commands in the script, this was just to get you started 😅